### DISCUSSION TITLE
Logs Panel: color highlighting of distinctive vs common words in similar loglines

### DISCUSSION THREAD

**Discussion by Dieterbe** [grafana/grafana#67973] [Dashboards, Panels & Visualizations]
Hello,
I have a sequence of log lines in loki that i visualize in Explore. These log lines are all quite similar.  Deduplication by number of signature reduces them to 1.  But the deduplication process is quite opaque: it doesn't tell me which numbers were identical across lines, which were different, and what their values were.

I would like to still see all my log lines, but have color highlighting applied to them, such that unique/distinctive words or numbers within them are highlighted one color, and words/sequences of words that are identical across the lines are highlighted in another color.
Perhaps even applying a color scale, based on how the common the text is across all lines in view.

thanks.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82685. Feel free to continue the discussion around this there. Thank you!
